units/src/amount/signed.rs

```diff
@@ -96,7 +123,12 @@ impl SignedAmount {
     /// per bitcoin overflows an `i64` type.
     pub const fn from_int_btc_const(btc: i64) -> SignedAmount {
         match btc.checked_mul(100_000_000) {
-            Some(amount) => SignedAmount::from_sat(amount),
+            Some(amount) => {
+                match SignedAmount::from_sat(amount) {
+                    Ok(sa) => sa,
+                    Err(_) => panic!("checked_mul overflow")
```

Comment:
This code does not panic because of checked_mul overflow.

Reply:
whoops, fixed.  Thanks!